WordPress/src/main/java/org/wordpress/android/ui/main/WPMainTabLayout.java

```diff
@@ -64,8 +63,8 @@ private void addTab(@DrawableRes int iconId, @StringRes int contentDescriptionId
      * unread notifications
      */
     void checkNoteBadge() {
-        showNoteBadge(SimperiumUtils.hasUnreadNotes());
-    }
+        showNoteBadge(false);
```

Comment:
It's not that easy because we're setting it to false in `WPMainActivity.java` https://github.com/wordpress-mobile/WordPress-Android/commit/b5dd0f28ba7b6e2a9270b6ec53f298f390b6f31d regardless of the count status.
We're doing it even with Simperium in. Don't know if it was by design.

I've updated the TODO - so we can on solution for this issue. Also opened a ticket for it in the master thread.